enterprise/frontend/src/embedding-sdk/docs/README.md

```diff
@@ -0,0 +1,197 @@
+
+```jsx
+// You can provide a custom action with your own `onClick` logic.
+const createCustomAction = clicked => ({
+  buttonType: "horizontal",
+  name: "client-custom-action",
+  section: "custom",
+  type: "custom",
+  icon: "chevronright",
+  title: "Hello from the click app!!!",
+  onClick: ({ closePopover }) => {
+    alert(`Clicked ${clicked.column?.name}: ${clicked.value}`);
+    closePopover();
+  },
+});
+
+// Or customize the appearance of the custom action to suit your need.
+const createCustomActionWithView = clicked => ({
+  name: "client-custom-action-2",
+  section: "custom",
+  type: "custom",
+  view: ({ closePopover }) => (
+    <button
+      className="tw-text-base tw-text-yellow-900 tw-bg-slate-400 tw-rounded-lg"
+      onClick={() => {
+        alert(`Clicked ${clicked.column?.name}: ${clicked.value}`);
+        closePopover();
+      }}
+    >
+      Custom element
+    </button>
+  ),
+});
+
+const plugins = {
+  /**
+   * You will have access to default `clickActions` that Metabase render by default.
+   * So you could decide if you want to add custom actions, remove certain actions, etc.
+   */
+  mapQuestionClickActions: (clickActions, clicked) => {
+    return [
+      ...clickActions,
+      createCustomAction(clicked),
+      createCustomActionWithView(clicked),
+    ]
+  }
+}
+
+return (
+  <MetabaseProvider config={config} pluginsConfig={plugins}>
+    <InteractiveQuestion questionId={firstQuestion.id}  />
+  </MetabaseProvider>
+);
+```
+
+After you have configured Metabase embedding SDK for React, you need to make sure your `jwtProviderUri` endpoint returns JWT token that your Metabase instance can use for authentication.
+
+**Metabase embedding SDK for React only support JWT authentication.**
+
+Here is the example how you could implement an endpoint that will return JWT token that Metabase embedding SDK for react could use to authenticate your users.
+
+```ts
+import express from "express"
+import type { Request, Response } from "express"
+
+import jwt from "jsonwebtoken"
+import fetch from "node-fetch"
+
+async function metabaseAuthHandler(req: Request, res: Response) {
+  const { user } = req.session
+
+  if (!user) {
+    return res.status(401).json({
+      status: 'error',
+      message: 'not authenticated',
+    })
+  }
+
+  const token = jwt.sign(
+    {
+      email: user.email,
+      first_name: user.firstName,
+      last_name: user.lastName,
+      groups: [user.group],
+      exp: Math.round(Date.now() / 1000) + 60 * 10, // 10 minutes expiration
+    },
+    // This is the JWT signing secret in your Metabase JWT authentication setting
+    METABASE_JWT_SHARED_SECRET
+  )
+  const ssoUrl = `${METABASE_INSTANCE_URL}?token=true&jwt=${token}`
+
+  try {
+    const response = await fetch(ssoUrl, { method: 'GET' })
+    const text = await response.text()
+    const token = JSON.parse(text)
```

Comment:
```suggestion
    const token = await response.json()
```

I did an additional check in the customer-zero demo to add a warning in case of instances with JWT auth disabled, but we don't have this check here - so we can directly convert to JSON :)